modules/touch/src/touchinteraction.cpp

```diff
@@ -390,37 +348,42 @@ void TouchInteraction::updateStateFromInput(const std::vector<TouchInputHolder>&
     }
 
     if (_tap) {
-        // check for doubletap
+        // @TODO (2023-02-01, emmbr) This if is not triggered on every touch tap.
```

Comment:
Could it be the `_tap = false` at the bottom of the `step` function?

Reply:
Nope, that's not it. But the logic for the tap depends on a huge check when removing a finger from the touch screen, so I suspect something might go wrong there. 

https://github.com/OpenSpace/OpenSpace/blob/d7eec159fc707c577c9632113843c1fb72d879c8/modules/touch/touchmodule.cpp#L249

Reply:
Leaving this in there for now; future me's (or someone else's) problem